content/docs/engine-dev/assets/metadata.md

```diff
@@ -0,0 +1,104 @@
+---
+linkTitle: Metadata Relocation
+title: Metadata Relocation
+description: Classes that facilitate metadata relocation system
+weight: 100
+---
+
+## What it is
+
+Metadata relocation allows files to be moved/renamed without breaking references.  It stores the UUID for a source asset in a .meta file alongside the source.  By moving/renaming the metadata file, Asset Processor (AP) can always keep track of the UUID.

```

Comment:
Because this is the maintainer documentation, it might be wroth both a a brief "Why is this system needed?" section up here, and then a "Why asset ID stabilization via sidecar files, over other asset relocation solutions?"

For the "Why is this system needed?", you could include a brief description like:

> Content creators often want to rename and move files, to improve project organization, and to fix typos. However, there are challenges with renaming and moving files: content tends to reference other content, and renaming or moving one file can potentially break incoming references to that file, from other files. Metadata relocation allows for the stabilization of asset identification via a UUID that remains the same as a file is moved and renamed, so that the file references can automatically handle that file operation.

For "Why asset ID stabilization via sidecar files, over other asset relocation solutions?" you could include something like:
>  Asset ID stabilization via sidecar files provides a low friction workflow for asset relocation, that can be done outside of O3DE tools, including with the source control tools that teams are probably already using. Other solutions either involve leaving bread crumb files or bread crumb information that can become a long term maintenance headache, or attempting to do a reference fix up every time a file is relocated or renamed, requiring custom tools to manage that reference fix up. In both cases, content creators may still move a file without using those systems, breaking references. Sidecar files are visible and discoverable, so content creators are more likely to move assets with the sidecar files, keeping references between assets in a project more stable, long term.

When I'm reading engine maintainer docs, I want to know both the intent behind the design as much as the design itself. It helps me understand how to safely make changes to that system that match that intended design.

Reply:
Good suggestion, I've added both of these